episodes/02-setup.md

```diff
@@ -213,12 +426,37 @@ $ git help
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-[git-privacy]: https://help.github.com/articles/keeping-your-email-address-private/
+-->
+
+::::::::::::::::: instructor
+
+When using `usethis::git_sitrep()`, check if there is no `✖ ...` line in the output with an error message. 
+
+Some common errors are solved here:
+
+- [Happy Git with R ebook](https://happygitwithr.com/https-pat.html#pat-troubleshooting)
+- [Workbench documentation](https://carpentries.github.io/sandpaper-docs/github-pat.html#common-token-errors)
 
+:::::::::::::::::::::::::
+
+::::::::::::::::: callout
+
+When using the [Terminal](https://glosario.carpentries.org/en/#console), this steps is known as `git config` for `--global` option. In the next chapter we are going to interact with the Terminal!
+
+:::::::::::::::::::::::::
+
+<!--
+If you are interested on this, [take a look to this chapter of the happygitwithr ebook](https://happygitwithr.com/hello-git).
+-->
+
+[git-privacy]: https://help.github.com/articles/keeping-your-email-address-private/
 
 :::::::::::::::::::::::::::::::::::::::: keypoints
 
-- Use `git config` with the `--global` option to configure a user name, email address, editor, and other preferences once per machine.
+- Use the `{usethis}` package to configure a user name, email address, and other preferences once per machine.
+- Use `usethis::use_git_config()` to configure git in Rstudio.
```

Comment:
You are mostly using Git as a proper noun (i.e. capitalising the G), but not always, I think either is fine, but be consistent.

Reply:
Thanks, I skimmed the docu that was the only one that needed an edit. https://github.com/epiverse-trace/git-rstudio-basics/pull/13/commits/bd5d2f0b72f1f882c549b1b9907c937e14b6a5ef